content/en/docs/contribute/new-content/new-features.md

```diff
@@ -0,0 +1,139 @@
+---
+title: Documenting a new feature
+linktitle: New features
+content_template: templates/concept
+main_menu: true
+weight: 20
+card:
+  name: contribute
+  weight: 45
+  title: Documenting a new feature
+---
+{{% capture overview %}}
+
+Each major Kubernetes release includes new features.
+Most of these features need documentation to show people how to use them.
```

Comment:
Line 15, 
Each major Kubernetes release introduces new features that require documentation. New releases also bring updates to existing features and documentation (such as upgrading a feature from alpha to beta).